include/macroni/Conversion/Safety/SafetyRewriters.hpp

```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+#include <macroni/Dialect/Macroni/MacroniDialect.hpp>
+#include <macroni/Dialect/Macroni/MacroniOps.hpp>
+#include <macroni/Dialect/Safety/SafetyDialect.hpp>
+#include <macroni/Dialect/Safety/SafetyOps.hpp>
+#include <mlir/Transforms/DialectConversion.h>
+#include <vast/Dialect/HighLevel/HighLevelOps.hpp>
+
+namespace macroni::safety {
+    using PR = mlir::PatternRewriter;
+
+    using IO = vast::hl::IfOp;
+    mlir::LogicalResult rewrite_unsafe(IO if_op, PR &rewriter);
```

Comment:
Those type aliases are polluting the whole namespace (as they are in a header) and do not help me at all if I want to see what to pass to `rewrite_unsafe`. Remove.